api/SVGTransformList.json

```diff
@@ -48,6 +48,342 @@
           "deprecated": false
         }
       },
+      "appendItem": {
+        "__compat": {
+          "spec_url": "https://svgwg.org/svg2-draft/types.html#__svg__SVGNameList__appendItem",
```

Comment:
This looks wrong but is right, `SVGNameList` is a sort of template for lots of similar interfaces.